### ISSUE TITLE
expand_locations crashes if `tool_prefix` prefix of `make` rule is set

### ISSUE THREAD

**Issue by fwingerter-Ocient** [bazelbuild/rules_foreign_cc#745]
[This line](https://github.com/bazelbuild/rules_foreign_cc/blob/main/foreign_cc/make.bzl#L57) seems incorrect (it should have an initial `ctx` argument), and gives the following backtrace:
```
Traceback (most recent call last):
	File "rules_foreign_cc/foreign_cc/make.bzl", line 36, column 33, in _make
		return cc_external_rule_impl(ctx, attrs)
	File "rules_foreign_cc/foreign_cc/private/framework.bzl", line 398, column 38, in cc_external_rule_impl
		] + attrs.create_configure_script(ConfigureParameters(ctx = ctx, attrs = attrs, inputs = inputs)) + postfix_script + [
	File "rules_foreign_cc/foreign_cc/make.bzl", line 57, column 43, in _create_make_script
		prefix = "{} ".format(expand_locations(attrs.tool_prefix, data)) if attrs.tool_prefix else ""
	File "rules_foreign_cc/foreign_cc/private/framework.bzl", line 894, column 5, in expand_locations
		def expand_locations(ctx, expandable, data):
Error: expand_locations() missing 1 required positional argument: data
```

Using 0.5.0 release .tar.gz.

**Comment by UebelAndre** [comment]
Hey! Yes, that's an unfortunate miss 😞 I'd be happy to review a PR fixing the missing `ctx` if this is blocking you. Otherwise, this revealed a clear gap in our test coverage that we'll have to fix but that'll take a little bit longer...

**Comment by UebelAndre** [comment]
The fix for this should be available in release [0.5.1](https://github.com/bazelbuild/rules_foreign_cc/releases/tag/0.5.1)
